source/jade/_mixins/user-mixins.jade

```diff
@@ -101,7 +101,7 @@ mixin alert-pwd-sent
 mixin alert-sign-up-failed
   if viewModel && viewModel.errorMessage
     .alert(role='alert')
-      #message= 'Error: ' + viewModel.errorMessage
+      #message!= 'Error: ' + viewModel.errorMessage
```

Comment:
@luchia can you add a mixin for TM support email? so that we can write

```jade
#message= 'Error: ' + viewModel.errorMessage
+tm-support-email
```